.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  validation:
+    name: Validation
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.1'
```

Comment:
I dont think there is a point in trying to keep it 7.1+ since we are now 7.3+ (and actually even 7.4+ since this week).
Best to use 7.3 from here on

Reply:
Fixed